src/flint/fmpq_mpoly.jl

```diff
@@ -789,6 +789,37 @@ f values")
    return r
 end
 
+function evaluate(a::fmpq_mpoly, bs::Vector{fmpq_mpoly})
+   R = parent(a)
+   S = parent(bs[1])
+
+   length(bs) != nvars(R) &&
+      error("Number of variables does not match number of values")
+
+   c = S()
+   fl = ccall((:fmpq_mpoly_compose_fmpq_mpoly, libflint), Cint,
+              (Ref{fmpq_mpoly}, Ref{fmpq_mpoly}, Ptr{Ref{fmpq_mpoly}},
+               Ref{FmpqMPolyRing}, Ref{FmpqMPolyRing}),
+              c, a, bs, R, S)
+   fl == 0 && error("Something wrong in evaluation.")
+   return c
+end
+
+function evaluate(a::fmpq_mpoly, bs::Vector{fmpq_poly})
+   R = parent(a)
+   S = parent(bs[1])
+
+   length(bs) != nvars(R) &&
+      error("Number of variables does not match number of values")
+
+   c = S()
+   fl = ccall((:fmpq_mpoly_compose_fmpq_poly, libflint), Cint,
+              (Ref{fmpq_poly}, Ref{fmpq_mpoly}, Ptr{Ref{fmpq_poly}},
+               Ref{FmpqMPolyRing}), c, a, bs, R)
+   fl == 0 && error("Something wrong in evaluation.")
+   return c
+end
```

Comment:
This whole thing seems to be a duplicate. ditto for ZZ

Reply:
Duplicate of what? I did not get an "method overwritten" error.

Reply:
My bad